textx/model.py

```diff
@@ -289,6 +297,62 @@ def get_model_from_str(self, model_str, file_name=None, debug=None,
                 pass
             return model
 
+        # Custom attr dunder methods used for user classes during loading
+        def _hasattr(obj, name):
+            try:
+                obj_attrs = obj.__class__._tx_obj_attrs
+                return name in obj_attrs[id(obj)]
+            except AttributeError:
+                return False
+
+        def _getattr(obj, name):
+            try:
+                return obj._tx_obj_attrs[id(obj)][name]
+            except KeyError:
+                raise AttributeError
```

Comment:
Some objects of this class may be fully initialized ordinary objects (for example builtins), so we should fall back to the original method if the id is not in this set.